brave/brave-core
# Fix Reading list graphics

Resolves https://github.com/brave/brave-browser/issues/24927

<img width="578" alt="image" src="https://user-images.githubusercontent.com/2965009/225312516-55de8262-19e2-4ff4-bec4-ef039adb9550.png">



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- open sidebar
- check bookmark page icon
